File: src/main/java/com/bajdcc/LALR1/interpret/os/task/TKProc.java
package com.bajdcc.LALR1.interpret.os.task;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【服务】进程管理
 *
 * @author bajdcc
 */
public class TKProc implements IOSCodePage {
	@Override
	public String getName() {
		return "/task/proc";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/task/TKProc.kt
package com.bajdcc.LALR1.interpret.os.task

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【服务】进程管理
 *
 * @author bajdcc
 */
class TKProc : IOSCodePage {
    override val name: String
        get() = "/task/proc"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
